### DISCUSSION TITLE
Thank you for your contributions (November 2023)

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11579] [Community]
We'd like to express our sincere appreciation to @thesujai, @KshitijThareja, @a6ar55, @Tweniee, @dbnicholson and @ShivangRawat30. Thank you for your wonderful contributions to the Kolibri ecosystem in November. In addition to this message, we're planning to mention all of you in our next [Community Newsletter](https://learningequalitynewsletter.getresponsepages.com/).

**Comment by Tweniee** [comment]
You're welcome! I appreciate the recognition, and I'm glad to contribute to the Kolibri ecosystem.

**Comment by vkWeb** [comment]
Thank you all, every line of code help us move forward :heart:

**Comment by carinediaz** [comment]
Thanks, everyone, for all your incredible contributions. They mean the world to us! We couldn't do it without you and truly appreciate the help improving Kolibri for all learners and educators out there!

**Comment by nucleogenesis** [comment]
I'm so deeply grateful for all of your contributions to our work! Thank you all so much for everything <3

**Comment by 0xV0YD** [comment]
@MisRob  hello mam i would like to contribute now again as the time i was off , i learned about django , app dev and now i am ready to indulge fully in open source contribution can i start from setting up the app locally on my pc ?

**Reply by MisRob** [reply]
Hi @yash1378, I linked contributing guidelines in the other thread you posted.

**Reply by 0xV0YD** [reply]
sorry for the different messages i will adhere to the guidelines from now on

**Reply by MisRob** [reply]
No worries
